dlang/source/flatbuffers/flatbufferbuilder.d

```diff
@@ -0,0 +1,470 @@
+module flatbuffers.flatbufferbuilder;
+
+import flatbuffers.exception;
+import flatbuffers.bytebuffer;
+
+import std.exception;
+import std.traits : isNumeric;
+
+final class FlatBufferBuilder
```

Comment:
This should build an inputRange, so that you don't need to take care of the memory management